api-docs/client-interactions.md

```diff
@@ -0,0 +1,198 @@
+    },
+    {
+      "id": 2,
+      "title": "Financial Counseling - Financial Coach Referal",
+      "interactionType": "in person",
+      "notes": "Appointment setup with financial coach. Client would like to save up enough for a down payment on a car better suited for their work.",
+      "startTime": "2019-05-06T05:00:00.000Z",
+      "endTime": "2019-05-06T06:00:00.000Z",
+      "location": "CU Office",
+      "referenceServices": ["DACA", "Youth Group"],
+      "canModify": true,
+      "isDeleted": false,
+      "createdBy": {
+        "userId": 1,
+        "firstName": "Joel",
+        "lastName": "Denning",
+        "fullName": "Joel Denning",
+        "timestamp": "2019-05-06T06:00:00.000Z"
+      }
+    }
+  ]
+}
+```
+
+## Create a Client Interaction
+
+### Request
+
+```http
+POST /api/clients/:id/interactions
+```
+
+```json
+{
+  "title": "Financial Counseling - Financial Coach Referal",
+  "interactionType": "in person",
+  "notes": "Appointment setup with financial coach. Client would like to save up enough for a down payment on a car better suited for their work.",
+  "startTime": "2019-05-06T05:00:00.000Z",
+  "endTime": "2019-05-06T06:00:00.000Z",
+  "location": "CU Office",
+  "referenceServices": ["DACA", "Youth Group"],
+  "canModify": true,
+  "isDeleted": false,
+  "createdBy": {
+    "userId": 1,
+    "firstName": "Joel",
+    "lastName": "Denning",
+    "fullName": "Joel Denning",
+    "timestamp": "2019-05-06T06:00:00.000Z"
+  }
+}
+```
+
+**_Notes_**
+
+- `program` is an enum with the following possible string values `Preventive Health - VDS daily services`, `Preventive Health - VDS mobile`, `Preventive Health - VDS Sabath`, `Preventive Health - Mobile Health Services`, `Preventive Health - Other`, `Nutrition - Nutrition`, `Nutrition - Supermarket visit`, `Nutrition - Cooking class`, `Nutrition - SNAP`, `Immigration - Citizenship`, `Immigration - Family Petition`, `Immigration - DACA`, `Immigration - Other`, `Financial Counseling`, `Worker Rights`, `Youth Group`, `Community Organization - Volunteering`, `Community Organization - Promoter`, `Community Organization - Internship`, `Family Support Services - LDS Vouchers`, `Family Support Services - Faxes`, `Family Support Services - References outside of CU`, `Family Support Services - Other`
+
+- `interactionType` is an enum with the possible values of `in person`, `by phone`, `workshop/talk`, `one on one/light touch`, `consultation`
+
+- `referenceServices` is an enum with the possible values of `citizenship`, `family petition`, `workers' rights/safety`, `DACA`, `youth group`, `promoters/leadership classes`, `SNAP/food stamps`, `evidence of chronic diseases`, `nutrition`, `supermarket visit`, `food demonstartion`, `community organization/advocacy`, `financial education/advising`, `financial coach`, `family support services`, `outside referral`, or `other`
+
+### Response
+
+The response object is exactly the same as the response object inside of the client log.
```

Comment:
"same response object inside of the client log" is wrong. Client log objects don't have the following properties:

- `notes`
- `startTime`
- `endTime`
- `location`
etc

Did you mean that this is the same object as when you do `GET /api/clients/:id/interactions/:id`?

Reply:
Change made last night